gettsim/policy_environment.py

```diff
@@ -272,7 +274,7 @@ def _parse_kinderzuschl_max(date, params):
 
     """
 
-    if date.year >= 2021:
+    if date.year >= 2024 and 2023 > date.year >= 2021:
         assert {"kinderzuschl", "kindergeld"} <= params.keys()
         params["kinderzuschl"]["maximum"] = (
             params["kinderzuschl"]["existenzminimum"]["regelsatz"]["kinder"]
```

Comment:
Problem is that we currently don't have the parameters `params["kinderzuschl"]["existenzminimum"]` implemented for now. I assume they do not exist yet? Then we should at least make a `ToDo:` note in the docstring of the function `_parse_kinderzuschl_max`

Reply:
Are those not the parameters in https://github.com/iza-institute-of-labor-economics/gettsim/blob/4379a90b5639e64628a5cb14cd411b9225b12240/gettsim/parameters/kinderzuschl.yaml#L89 ? They might be outdated for 2023 and 2024. I could update them as part of this PR (but would prefer to do so in a separate PR). For KiZ they should only become relevant (again) in 2024 (we will see, probably not).

Reply:
I updated these parameters in #446. But I don't think this PR really depends on that being done first.